.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+<!--
+Thanks for submitting a change!
+Before opening this pull request, please fill out the following checklist:
+
+NOTE: If you tick an item with **API review required** please tag the @openshift/api-reviewers group and wait until member of this group review this pull request before merging.
+-->
+
+<!-- WRITE DESCRIPTION HERE -->
+
+---
+
+#### API change checklist
+
+- [ ] This change introduces a new API or changes an existing API in a way that is complaint with the [upstream API conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md)  _(**API review required**)_
```

Comment:
and OpenShift ones https://github.com/openshift/enhancements/blob/master/CONVENTIONS.md#api as well they are covering certain areas that are openshift specific and not covered by upstream